dev/conductor/core/lib/src/globals.dart

```diff
@@ -171,7 +196,8 @@ String getNewPrLink({
       repoLabel = 'Engine';
       break;
     default:
-      throw ConductorException('Expected repoName to be one of flutter or engine but got $repoName.');
+      throw ConductorException(
```

Comment:
These line fixes seem fine to me, but I defer to @christopherfujino 

I prefer codebases with 120 line lengths, but I do see that the framework generally requires 80 (https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters)

Reply:
I defer to Casey :)

Reply:
This was due to VSCode linting.  Maybe I need to change my linter settings. Thanks for the heads up!

Reply:
Yeah, by default the Flutter/dart plugin will call dart format on save, which is against flutter style's guide.